src/site/_includes/components/navigation/index.scss

```diff
@@ -12,6 +12,21 @@ $navigation-height: 50px;
   }
 }
 
+@mixin moduk-maroon {
```

Comment:
Maybe we could define this centrally and use it for the `.moduk-link--maroon` class as well.

Reply:
I'll place it in the global folder